src/genophenocorr/view/_protein_viewer.py

```diff
@@ -0,0 +1,48 @@
+import typing
+
+from jinja2 import Environment, PackageLoader
+from collections import OrderedDict
+
+from genophenocorr.model import Cohort
+from genophenocorr.model.genome import Region
+from ._protein_visualizable import ProteinVisualizable
+
+
+class ProteinViewable:
+    """
+    TODO
+    """
+    def __init__(self) -> None:
+        environment = Environment(loader=(PackageLoader('genophenocorr.view', 'templates')))
+        self._cohort_template = environment.get_template("protein.html")
+        
+    def process(self, cohort: Cohort, pvis: ProteinVisualizable) -> str:
+        context = self._prepare_context(cohort, pvis)
+        return self._cohort_template.render(context)
+    
+    @staticmethod
+    def _get_start(feat_list: typing.Iterable[typing.Any]) -> int:
+        return feat_list[2].start
+    
+    def _prepare_context(self, cohort: Cohort, pvis: ProteinVisualizable) -> typing.Mapping[str, typing.Any]:
+        protein_id = pvis.protein_id
+        protein_features = []
+        
+        for i in range(len(pvis.protein_feature_names)):
+            protein_features.append([pvis.protein_feature_names[i], pvis._protein_feature_types[i], Region(pvis.protein_feature_starts[i],pvis.protein_feature_ends[i]), 0])
```

Comment:
Right now, `protein_features` is a list of lists and it takes a significant amount of brain power to understand what does it do.

Can you please make `protein_features` a list of named tuples? If you do not know what a named tuple is, please look it up in Python's documentation. Then create a named tuple that is private to this module and make a tuple for each list item. Then use the named tuple's properties in the downstream code.

The code is probably correct, but with named tuples you will make it easier to maintain and linters (and Python interpreter FWIW) will stand by you and help you find mistakes.